src/Client.cpp

```diff
@@ -151,23 +164,20 @@ int UrsaClient::start() {
 
     std::thread status_th(&UrsaClient::status_worker, this);
 
+    std::string prompt{is_interactive ? "ursadb>" : ""};
```

Comment:
"ursadb> "

Reply:
:thinking: yes, looks like the readline compatibility wrapper is crap